### PR TITLE
RetryLink

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: web/src/sdk/links/networkLink.js [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-                console.log(`[Network error]: ${networkError}`);
-                // if you would also like to retry automatically on
-                // network errors, we recommend that you use
-                // apollo-link-retry
-            }
-        });
+export const networkLink = () => (
+  onError(({graphQLErrors, networkError, operation, forward}) => {
+    if (networkError) {
+      // TODO check if mutation
```
Comment (wtrocki) [inline web/src/sdk/links/networkLink.js:6]: [Optional] Cleanup comments to reflect current state.

File: web/src/sdk/links/retryLink.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {RetryLink} from "apollo-link-retry";
+
+export const retryOnErrorLink = (numOfAttempts, firstAttempAfter) => (
+  new RetryLink({
+    delay: {initial: firstAttempAfter || 1000},
+    attempts: (count, operation, error) => {
+      console.log(`retryOnErrorLink: {
```
Comment (wtrocki) [inline web/src/sdk/links/retryLink.js:7]:
```suggestion
     console.info(`retryOnErrorLink: {    
```

**Review by StephenCoady — APPROVED** [review]
Just trying this out. Not sure if it is related to this PR (first guess is no) but when I start everything for the first time all mutations fail with an error about the version missing. Possibly cache related. Will approve this anyway and look for the fix separately.

**Comment by wtrocki** [comment]
@StephenCoady It could be that we save data to cache without version (version may be still missing or cache was old)

**Comment by wtrocki** [comment]
Let;s get that into master and work on this

**Pull request merged**